File: Arcade/Intro/Edge of the Ocean/CplusPlus/Make Array COnsecutive 2.cpp
int makeArrayConsecutive2(std::vector<int> statues)
{
    int a = statues.size();
    int diff = 0;
    int sum = 0;

    sort(statues.begin(), statues.end());

    for (int i = a - 1; i > 0; i--)
    {
        sum += (statues[i] - statues[i - 1]) - 1;
    }
    return sum;
}

File: Arcade/Intro/Edge of the Ocean/Javascript/Make Array Consecutive 2.js
function makeArrayConsecutive2(statues) {
  var a = statues.length;
  var diff = 0;
  var sum = 0;

  statues.sort(function (a, b) {
    return a - b;
  });

  for (var i = a - 1; i > 0; i--) {
    sum += statues[i] - statues[i - 1] - 1;
  }
  return sum;
}
